modules/tinymce/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Dialogs will not exceed the window height on smaller screens #TINY-8146
 - Text alignment could not be applied to `pre` elements #TINY-7715
+- Nuget packages for .NET and .NET Core now copy TinyMCE into /wwwroot/lib/ in the projects TinyMCE is installed in #TINY-8611
```

Comment:
```suggestion
- Nuget packages for .NET and .NET Core now copy TinyMCE into `/wwwroot/lib/` in the projects TinyMCE is installed in #TINY-8611
```
Also should this be in the `Improved` section instead? Not sure whether to class it as a bug or an enhancement to be honest. A bug is probably more correct.

Reply:
I was thinking the same thing. On the one hand, it does _improve_ the experience for users of the nuget package. On the other hand, it does _fix_ the fact that the package previously didn't put the files in the place that web project was expecting to find static content. I'm happy to leave it as `Fixed`.